importexport/capella/capella.cpp

```diff
@@ -206,11 +206,15 @@ static void processBasicDrawObj(QList<BasicDrawObj*> objects, Segment* s, int tr
                                           case 'u':   // fermata up
                                                 addArticulationText(score, cr, track, SymId::fermataAbove);
                                                 break;
+                                          case 'n':   // segno coda
+                                                addArticulationText(score, cr, track, SymId::coda);
+                                                break;
+                                          case 'y':   // segno
+                                                addArticulationText(score, cr, track, SymId::segno);
```

Comment:
A Segno isn't an articulation but a `Marker`

Reply:
Got it